k8s-service-account-config-to-ssm/kube_config_generator.py

```diff
@@ -0,0 +1,276 @@
+import base64
+import boto3
+import getopt
+import sys
+import re
+import subprocess
+import logging
+from kubernetes import client, config
+from kubernetes.client.rest import ApiException
+
+# define constants
+AWS_ROLE_CLOUD_ADMIN = "arn:aws:iam::738063116313:role/CloudAdmin"
```

Comment:
This is a public repo, so I don't think we should store security related data here. Instead it should be kept local in a .env file and loaded at runtime, for instance using this method: https://github.com/dfds/ce-toolbox/blob/master/kafka-delete-topic/delete_topics.py#L10

Reply:
An excellent point.  I will look into implementing this.